x/staking/types/validator.go

```diff
@@ -444,7 +445,6 @@ func (v *Validator) MinEqual(other *Validator) bool {
 		v.Description.Equal(other.Description) &&
 		v.Commission.Equal(other.Commission) &&
 		v.Jailed == other.Jailed &&
-		v.MinSelfDelegation.Equal(other.MinSelfDelegation) &&
```

Comment:
Should this still be removed now that the field was added back as deprecated? Same question for all the places where this field was removed.

Reply:
two cents from me: the field was marked as deprecated (as opposed to complete removal from the proto) just to fix proto decoder that will break otherwise, and will not be able to parse old state and txns. The field shouldn't be used in any code logic and assumed to be always 0.

So it's valid to ignore it there.